deployments/docker/distributed/docker-compose.yml

```diff
@@ -37,8 +37,8 @@ services:
       - milvus
 
   master:
-    image: registry.zilliz.com/milvus-distributed/milvus-distributed:PR-5020-release
-    command: ["/milvus-distributed/bin/milvus", "run", "master"]
+    image: registry.zilliz.com/milvus-distributed/milvus-distributed:master-release
```

Comment:
Image url still use milvus-distributed?

Reply:
This is a temporary Docker image that will be modified after release milvus .